netmaster/main.go

```diff
@@ -23,12 +23,14 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/contiv/netplugin/netmaster/daemon"
+	"github.com/contiv/netplugin/netmaster/docknet"
 	"github.com/contiv/netplugin/version"
 )
 
 type cliOpts struct {
 	help         bool
 	debug        bool
+	pluginName   string
```

Comment:
netmaster needs to know the plugin name so that it can use it in the docknet apis to create the docker network using the proper plugin name